conans/client/command.py

```diff
@@ -224,23 +224,38 @@ def package_files(self, *args):
                                          profile_name=args.profile, force=args.force,
                                          settings=args.settings, options=args.options)
 
+    def download(self, *args):
+        """Downloads a conan package to the local cache without using settings by specifying the
```

Comment:
Clarify the purpose. Say that it is not transitive. Maybe the main use case is the "conan copy" to promote packages

Reply:
Done